### ISSUE TITLE
thresh is 0.6?

### ISSUE THREAD

**Issue by kevinchan04** [amdegroot/ssd.pytorch#494]
https://github.com/amdegroot/ssd.pytorch/blob/5b0b77faa955c1917b0c710d770739ba8fbff9b7/test.py#L64

**Comment by AtriSaxena** [comment]
Yes. Threshold here set to 60 %. You can change based on your problem.
